src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -1,100 +1,121 @@
+// eslint-disable-next-line no-use-before-define
 import * as React from 'react';
-import {Dimensions, StyleSheet, Text, View} from 'react-native';
+import ContextMenu from 'react-native-context-menu-view';
+import Icon from 'react-native-vector-icons/Entypo';
+import {Animated, Text, View} from 'react-native';
+import {Swipeable} from 'react-native-gesture-handler';
+import {useAnimatedStyle, useSharedValue} from 'react-native-reanimated';
 
+import ChatReplyView from './ChatReplyView';
+import useMessageHook from '../../../hooks/screen/useMessageHook';
 import {ChatItemMyTextProps} from '../../../../types/component/AnonymousChat/BaseChatItem.types';
 import {colors} from '../../../utils/colors';
-import {fonts} from '../../../utils/fonts';
+import {
+  containerStyle,
+  dotStyle,
+  messageStyle,
+  styles,
+  textContainerStyle,
+  textStyle
+} from './ChatItemText.style';
 
-const {width} = Dimensions.get('screen');
-const styles = StyleSheet.create({
-  chatContainer: {
-    display: 'flex',
-    flexDirection: 'row',
-    marginTop: 4,
-    marginBottom: 4,
-    maxWidth: width,
-    paddingRight: 60,
-    paddingLeft: 10
-  },
-  chatTitleContainer: {
-    display: 'flex',
-    flexDirection: 'row'
-  },
-  textContainer: {
-    backgroundColor: colors.lightgrey,
-    paddingLeft: 8,
-    paddingRight: 8,
-    paddingTop: 4,
-    paddingBottom: 8,
-    borderRadius: 8,
-    borderTopStartRadius: 0,
-    flex: 1
-  },
-  userText: {
-    fontFamily: fonts.inter[600],
-    fontSize: 12,
-    lineHeight: 19.36
-  },
-  text: {
-    fontFamily: fonts.inter[400],
-    fontSize: 16,
-    lineHeight: 19.36
-  },
-  avatar: {
-    width: 24,
-    height: 24,
-    borderRadius: 12,
-    marginRight: 8
-  },
-  dot: {
-    width: 3,
-    height: 3,
-    borderRadius: 2,
-    marginLeft: 5,
-    marginRight: 5,
-    backgroundColor: colors.black,
-    alignSelf: 'center'
-  },
-  timeText: {
-    fontFamily: fonts.inter[200],
-    fontSize: 10,
-    lineHeight: 12.19,
-    alignSelf: 'center'
-  },
-  mr8: {
-    marginRight: 8
-  }
-});
+export const replyIcon = () => {
+  return (
+    <View style={styles.containerReply}>
+      <View style={styles.containerReplyIcon}>
+        <Icon name={'reply'} size={20} color={colors.black} />
+      </View>
+    </View>
+  );
+};
 
 const ChatItemTargetText = ({
   username = 'Anonymous User',
   time = '4h',
   isContinuous = false,
   message = '',
-  avatar
+  avatar,
+  chatType,
+  messageType,
+  data
 }: ChatItemMyTextProps) => {
+  const {setReplyPreview, onContextMenuPressed} = useMessageHook();
+
+  const swipeableRef = React.useRef<Swipeable | null>(null);
+  const bubblePosition = useSharedValue(0);
+
+  const isReply = messageType === 'reply';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `swipeableRef` is defined but not used anywhere in the component. If it's not needed, consider removing it to clean up the code.
```diff
-  const swipeableRef = React.useRef<Swipeable | null>(null);
-  const bubblePosition = useSharedValue(0);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->